tee-worker/ts-tests/common/utils.ts

```diff
@@ -416,7 +419,7 @@ export async function checkUserShieldingKeys(
     };
     let resp = await sendRequest(context.tee, request, context.api);
 
-    return resp.value;
```

Comment:
Same here; I think it'd be safer to throw if `typeof resp.value !== 'string'` :thinking:

Reply:
I modified the usage of WorkerRpcReturnValue, it seems that we still need to manually define it here, as using the generated one is not good enough.

Reply:
Is it better to have this comment in the modified code? Otherwise, it might be confusing to read this part directly.Just a small friendly reminder